fieldpath/pathelementmap_test.go

```diff
@@ -29,22 +29,22 @@ func TestPathElementValueMap(t *testing.T) {
 		t.Fatal("Unexpected path-element found in empty map")
 	}
 
-	m.Insert(PathElement{FieldName: strptr("carrot")}, value.StringValue("knife"))
-	m.Insert(PathElement{FieldName: strptr("chive")}, value.IntValue(2))
+	m.Insert(PathElement{FieldName: strptr("carrot")}, value.ValueInterface{Value: "knife"})
```

Comment:
Can we get back the "StringValue()" family of functions? This change is unnecessary.